mk/cleandirs.mk

```diff
@@ -26,3 +26,19 @@ $(eval _O:=$(if $(O),$(O),.))$(wildcard $(addprefix $(_O)/,$(call _reverse,
 endef
 
 RMDIR := rmdir --ignore-fail-on-non-empty
+
+# Remove files with "rm -f".
+# Split (possibly huge) file list into more manageable lines
+# (200 files at a time), to minimize the odds of having:
+# "/bin/bash: Argument list too long"
+define do-rm-f
+        $(call _do-rm, $(wordlist 1, 200, $(1)))
```

Comment:
`_do-rm-f`?

Reply:
ditto

Reply:
Doh! That's what happens when you decide to rename something at the last minute.
Will fix, thanks.